src/core/js/device.js

```diff
@@ -187,10 +187,16 @@ define(function(require) {
         }
     }
 
-    // Convert 'msie' to 'ie' for backwards compatibility
-    var browserString = (Adapt.device.browser.toLowerCase() === 'msie') ? 'ie' : Adapt.device.browser.toLowerCase();
+    var browser = Adapt.device.browser.toLowerCase();
+    // Convert 'msie' to 'ie' for backwards compatibility.
+    var browserString = (browser === 'msie') ? 'ie' : browser;
```

Comment:
OK, apologies, I missed Tom's comment about disregarding the `internet explorer` class.